website/content/en/preview/upgrade/upgrade-ref.md

```diff
@@ -0,0 +1,606 @@
+| karpenter_machines_drifted                                            | karpenter_nodeclaims_drifted                                     |
+| karpenter_machines_initialized                                        | karpenter_nodeclaims_initialized                                 |
+| karpenter_machines_launched                                           | karpenter_nodeclaims_launched                                    |
+| karpenter_machines_registered                                         | karpenter_nodeclaims_registered                                  |
+| karpenter_machines_terminated                                         | karpenter_nodeclaims_terminated                                  |
+| karpenter_provisioners_limit                                          | karpenter_nodepools_limit                                        |
+| karpenter_provisioners_usage                                          | karpenter_nodepools_usage                                        |
+| karpenter_provisioners_usage_pct                                      | Dropped                                                          |
+| karpenter_deprovisioning_eligible_machines                            | karpenter_disruption_eligible_nodeclaims                         |
+| karpenter_deprovisioning_replacement_machine_initialized_seconds      | karpenter_disruption_replacement_nodeclaims_initialized_seconds  |
+| karpenter_deprovisioning_replacement_machine_launch_failure_counter   | karpenter_disruption_replacement_nodeclaims_launch_failed        |
+| karpenter_nodes_leases_deleted                                        | karpenter_leases_deleted                                         |
+
+In addition to these metrics, the MachineNotFound error returned by the `karpenter_cloudprovider_errors_total` values in the error label has been changed to `NodeClaimNotFound`. This is agnostic to the version of the API (Machine or NodeClaim) that actually owns the instance.
+
+## Global Settings
+
+The v1beta1 specification removes the `karpenter-global-settings` ConfigMap in favor of setting all Karpenter configuration using environment variables. Along, with this change, Karpenter has chosen to remove certain global variables that can be configured with more specificity in the EC2NodeClass . These values are marked as removed below.
+
+
+| **`karpenter-global-settings` ConfigMap Key**     | **Environment Variable**        | **CLI Argument**
+|---------------------------------------------------|---------------------------------|-----------------------------------|
+| batchMaxDuration                                  | BATCH_MAX_DURATION              | --batch-max-duration              |
+| batchIdleDuration                                 | BATCH_IDLE_DURATION             | --batch-idle-duration             |
+| aws.assumeRoleARN                                 | AWS_ASSUME_ROLE_ARN             | --aws-assume-role-arn             |
```

Comment:
We decided to remove the `aws` prefix from all of these CLI args and environment variables. Can you drop the aws prefix from all things in this table

Reply:
Sure. Done.